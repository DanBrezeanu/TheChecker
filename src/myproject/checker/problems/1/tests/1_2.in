-864692037
0
