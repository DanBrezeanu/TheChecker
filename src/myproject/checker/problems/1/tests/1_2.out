-864692037
-864692037
